index.html

```diff
@@ -74,173 +74,183 @@ <h2 class="works">My Recent<br> Works</h2>
+      you like what you see and have a<br> project you need coded,<br>don’t hesitate to contact me.</p></div>
+      <div><button class="button">Get my resume</button></div>
   </div>
 </div>
 
-<div class="griditem">
-    <div class=greyspace>
+</section>
+
+<section id=section4>
+  <div class="grid-container2">
+    <div class="griditem2">
       <div><img id="language" src="./Assets/icon-languages.svg" alt="Languages"></div>
       <h5>Languages</h5>
-          <ul class="minimenu">
+          <ul class="minimenu2">
             <li class="button3">Ruby on Rails</li>
             <li class="button3">CSS</li>  
             <li class="button3">Javascript</li>
             <li class="button3">HTML</li>    
           </ul>
-    </div>
-</div>
-
-<div class="griditem">
-  <div class=greyspace>
-    <div><img id="frameworks" src="./Assets/icon-frameworks.svg" alt="Icon"></div>
-    <h6>Frameworks </h6>
-        <ul class="minimenu">
+      </div>
+  
+    <div class="griditem2">
+      <div><img id="frameworks" src="./Assets/icon-frameworks.svg" alt="Icon"></div>
+      <h6>Frameworks </h6>
+        <ul class="minimenu2">
           <li class="button3">React.js</li>
           <li class="button3">Ruby on Rails</li>  
           <li class="button3">RSpec</li><br>      
-        </ul>
-        <ul class="minimenu">
           <li class="button3">Capybara</li> 
           <li class="button3">Selenium</li>       
         </ul>
-  </div>
-</div>
-
-<div class="griditem">
-  <div class=greyspace>
-    <div></div><img id="skills" src="./Assets/icon-skills.svg" alt="Skills"></div>
-    <h6>Skills</h6>
-        <ul class="minimenu">
+    </div>
+  
+    <div class="griditem2">
+    <div><img id="skills" src="./Assets/icon-skills.svg" alt="Skills"></div>
+      <h6>Skills</h6>
+        <ul class="minimenu2">
           <li class="button3">Database Management</li>
           <li class="button3">Version Control</li>   
-        </ul>
-
-        <ul class="minimenu"> 
           <li class="button3">CLI</li>
           <li class="button3">Web Developement</li>  
           <li class="button3">API Design</li>  
         </ul>
-  </div>
+    </div>
 </div>
-
 </section>
 
 </main>
 
-  <footer class="footer"></footer>
+<div class="wrapper">
+  <footer class="footer">
+  
+<div><p id=footertext>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
+  
+<div class="form">
+<form action="https://formspree.io/f/meqwblog" method="post">
+      <input type="text" id="name" name="user_name" placeholder="Full Name" maxlength="30">
+      <input type="email" id="mail" name="user_email" placeholder="Email Address" pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}" required>
+      <textarea id="msg" name="user_message" placeholder="Hey I'd like to connect to..." maxlength="500"></textarea>
```

Comment:
You have done a great job so far 👍. Kindly add the required attribute in the name and in the textarea as requested by the previous reviewer. This will help make sure your project meets the requirements and also will make it look more professional.

Reply:
Thank you for the feedback.

Reply:
> - [ ] Great progress so far! 👍 I wanted to bring to your attention a small oversight pointed out by the previous reviewer. It seems that the required attribute was not included in the textarea. Adding this attribute would be greatly appreciated as it ensures that your project aligns with the necessary requirements and provides an enhanced user experience.😊